clio-util/src/main/scala/org/broadinstitute/clio/util/auth/AuthUtil.scala

```diff
@@ -28,22 +40,34 @@ object AuthUtil extends ModelAutoDerivation with LazyLogging {
     serviceAccountPath: Option[Path]
   ): Either[Throwable, OAuth2Credentials] = {
 
-    val shellOutCreds: Either[Throwable, OAuth2Credentials] = Right(
-      ExternalGCloudCredentials
-    )
+    val shellOutCreds: Either[Throwable, OAuth2Credentials] = {
+      logger.info("Shelling out to gcloud to get credentials")
+      Right(ExternalGCloudCredentials)
+    }
 
-    serviceAccountPath.fold(shellOutCreds) { jsonPath =>
-      loadServiceAccountJson(jsonPath).fold(
-        err => {
-          logger.warn(
-            s"Failed to load service account JSON at $jsonPath, falling back to gcloud",
-            err
-          )
-          shellOutCreds
-        },
-        getCredsFromServiceAccount
+    // Have to go from try to either since either doesn't have an 'orElse' method
+    Try(GoogleCredentials.getApplicationDefault)
```

Comment:
This shouldn't be the top-priority option; it makes it impossible to override the credentials by passing service-account-json.